packages/1155-contracts/src/delegation/ZoraCreator1155PremintExecutorImpl.sol

```diff
@@ -212,35 +146,44 @@ contract ZoraCreator1155PremintExecutorImpl is
         return (true, ERC1155DelegationStorageV1(contractAddress).delegatedTokenId(uid));
     }
 
+    /// deprecated - use isValidSignatureV1
```

Comment:
Can you make this a natspec so it shows up on etherscan?

can also add `@custom:deprecated use isValidSignatureV1 instead`

https://github.com/OpenZeppelin/openzeppelin-contracts/issues/3642